base/bitarray.jl

```diff
@@ -94,7 +94,7 @@ julia> BitVector(nt)
 ```
 """
 function BitVector(nt::Tuple{Vararg{Bool}})
-    bv = BitVector(undef, length(nt))
+    bv = BitVector(undef, length(nt)::Int)
```

Comment:
Why is this one needed? Are there circumstances under which the `length` call here cannot be inferred as `::Int`? Then maybe that should be looked at.

Certainly we get
```julia
julia> code_typed(length, Tuple{Tuple{Vararg{Bool}}})
1-element Vector{Any}:
 CodeInfo(
1 ─ %1 = Base.nfields(t)::Int64
└──      return %1
) => Int64
```
So it can't be too bad.